api/settings.py

```diff
@@ -93,6 +95,10 @@ def _default(variable):
 with open(os.path.join(os.path.dirname(__file__), "version.txt")) as f:
     VERSION = f.read().strip()
 
+if CLOUDFRONT_PRIVATE_KEY:
+    with open(CLOUDFRONT_PRIVATE_KEY) as f:
```

Comment:
if you open this in binary, does that avoid having to encode it as utf-8 below?

Reply:
It does indeed, let's do that :)